src/components/Author.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+
+class Author extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      name: undefined,
+      email: undefined,
+      github: undefined
+    };
+  }
+
+  componentWillReceiveProps(nextProps) {
+    this.fetchAuthorData(nextProps.author);
+  }
+
+  componentDidMount() {
+    this.fetchAuthorData(this.props.author);
+  }
+
+  fetchAuthorData(author) {
+    let data = require(`../../gamedata/authors.json`).authors;
+    data = data[author];
+    if(!data) return;
+    this.setState({
+      name: data.name,
+      email: data.email,
+      github: data.github
+    });
+  }
+
+  render() {
+    const { name, email, github } = this.state;
+    const nodata = !name && !email && !github;
+    return (
+      <div>
+        <div style={{marginTop: '20px', marginBotton: '20px'}}>
+
+          Level author:
+          <br/>
+
+          {nodata && 
+            <span>{this.props.author}</span>
+          }
+
+          {!nodata && name &&
+            <span>{name}</span>
+          }
+
+          {!nodata && email && 
+            <span><br/><strong>{email}</strong></span>
```

Comment:
Nice to have, wrap it into a mailto link.

Reply:
Done.